Dockerfile

```diff
@@ -127,7 +127,6 @@ FROM base AS test
 RUN apt-get update && apt-get install -y \
   mariadb-client
 
-COPY Makefile Makefile
```

Comment:
Unrelated tidy up, this isn't needed as it's already copied further up the file.